overlay/3-node/kustomization.yaml

```
@@ -2,14 +2,14 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-namespace: blazer-cars-lab
-# namePrefix: blazer-cars-lab-
+namespace: volante-cars-lab
 
 resources:
-  - github.com/redhat-partner-solutions/vse-ocp-bases/base/3-node?ref=main
+  - ../../base/3-node
```

I think we should use the URL instead of relative path
```suggestion
  - github.com/redhat-partner-solutions/vse-ocp-bases/base/3-node?ref=main
```

---

you are right. I included there the relative path for testing purposes because it is independent of the branch you are in. However if we think about the "user" always better to use the URL :-)